### PR TITLE
Storage docs update

### PR DESCRIPTION
**Pull request by hdworld11**
### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [x] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] Swift
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://docs.amplify.aws/)` 
            HTML: `<a href="https://docs.amplify.aws/">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/pages/[platform]/build-a-backend/storage/download-files/index.mdx [hunk 1]
```diff
@@ -40,19 +42,187 @@ export const DefaultStorageImageExample = () => {
...
+```
+
+`getUrl` returns a signed URL in the `url` property of the result. You can use this to create a download link for users to click on. The `expiresAt` property is a `Date` object that represents the time at which the URL will expire.
+
+Inside your template or JSX code, you can use the `url` property to create a link to the file:
+
+```tsx
+<a href={signedURL.url.toString()} target="_blank" rel="noreferrer">
+  {fileName} 
+</a>
```
Comment (timngyn) [inline src/pages/[platform]/build-a-backend/storage/download-files/index.mdx:74]: Is this missing the closing backticks to close out the codeblock?

Comment (hdworld11) [reply src/pages/[platform]/build-a-backend/storage/download-files/index.mdx:74]: Yes, thanks for pointing it out. Updating it.

**Review by timngyn — DISMISSED** [review]

**Review by timngyn — APPROVED** [review]

File: src/pages/[platform]/build-a-backend/storage/upload-files/index.mdx [hunk 2]
```diff
@@ -45,15 +46,13 @@ export const DefaultStorageImageExample = () => {
 
 Learn more about how you can further customize the UI component by referring to the [Storage Manager documentation](https://ui.docs.amplify.aws/react/connected-components/storage/storagemanager).
 
-</Block>
-<Block name="Amplify Console">
+## Amplify Console
 You can also upload files ad-hoc via the Amplify Console once you have set up your backend.
 
 ![Amplify Console showing drag and drop feature to upload files on-demand](/images/gen2/storage/drag-and-drop.png)
```
Comment (renebrandel) [inline src/pages/[platform]/build-a-backend/storage/upload-files/index.mdx:52]: find and replace upper "C" with "c"

Comment (hdworld11) [reply src/pages/[platform]/build-a-backend/storage/upload-files/index.mdx]: Done, thanks!

**Review by timngyn — APPROVED** [review]

**Review by cwomack — APPROVED** [review]

File: src/pages/[platform]/build-a-backend/storage/upload-files/index.mdx [hunk 3]
```diff
@@ -26,13 +26,14 @@ export function getStaticProps(context) {
...
+## Storage Manager UI Component
+
 Upload files from your app in minutes by using the cloud-connected Storage Manager UI Component. 
 
 ```bash
 npm install @aws-amplify/ui-react-storage aws-amplify
 ```
+
 ```tsx
 import { StorageImage } from '@aws-amplify/ui-react-storage';
```
Comment (arundna) [inline src/pages/[platform]/build-a-backend/storage/upload-files/index.mdx:38]: Shouldn't this be importing `StorageManager` instead of `StorageImage`?

**Review by arundna — COMMENTED** [review]
Left comments on the upload-files section

File: src/pages/[platform]/build-a-backend/storage/download-files/index.mdx [hunk 4]
```diff
@@ -40,19 +42,188 @@ export const DefaultStorageImageExample = () => {
 
 Learn more about how you can further customize the UI component by referring to the [Storage Image documentation](https://ui.docs.amplify.aws/react/connected-components/storage/storagemanager).
```
Comment (arundna) [inline src/pages/[platform]/build-a-backend/storage/download-files/index.mdx:43]:
```suggestion
Learn more about how you can further customize the UI component by referring to the [Storage Image documentation](https://ui.docs.amplify.aws/react/connected-components/storage/storageimage).
```

File: src/pages/[platform]/build-a-backend/storage/upload-files/index.mdx (hunk 3 above)
Comment (hdworld11) [reply src/pages/[platform]/build-a-backend/storage/upload-files/index.mdx]: Thanks for pointing it out Arun, I have made the update. This code sample is also going to change and I will be including that in a different PR

**Review by aspittel — APPROVED** [review]

**Review by swaminator — APPROVED** [review]

**Pull request merged**